src/components/Loading.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from '../styles/loading'
+
+import React from 'react'
+
+const Loading = ({ message }) => {
+  return (
+    <div className={styles['fil-loading']}>
+      <p>{message}</p>
+    </div>
+  )
+}
+
+export default Loading
```

Comment:
It may use a translation's key instead of a string according to the other components, doesn't it?